mslice/models/slice/slice_functions.py

```diff
@@ -233,7 +234,7 @@ def _compute_powder_line_momentum(ws_name, q_axis, element, cif_file):
 def _crystal_structure(ws_name, element, cif_file):
     if cif_file:
         ws = get_workspace_handle(ws_name).raw_ws
-        run_algorithm('LoadCIF', store=False, InputWorkspace=ws, InputFile=cif_file)
```

Comment:
`run_algorithm` doesn't exist on the CLI branch. Everything is patched in `util.mantid_algorithms` instead. This was done so that mantid algorithms could be used with our workspace wrappers from the CLI without problems.

I think this `run_algorithm` call was accidentally added last time I merged master to CLI.

Admittedly, having two different ways to call algorithms depending on the branch is getting kind of confusing.

Reply:
Thanks for the explanation. Hopefully we won't have the split for too much longer so this won't cause much more confusion.